### PR TITLE
[PC-1435] First cut of model only, obj-c only Content Cards multiple feed docs

### PR DESCRIPTION
**Pull request by Bucimis**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)...

**Reason for Change:**
> I'm making this change because.....

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+```
+[[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(contentCardsUpdatedNotificationReceived:)
+                                               name:ABKContentCardsProcessedNotification
+                                             object:nil];
+```
+
+Add the following methods to respond to updates from the observer and filter the returned cards by type. 
+
+The first method, `contentCardsUpdatedNotificationReceived:`, handles updates from the observer. It the calls the second method, `getCardsForFeedType:`, with the desired feed type, in this case `Transactional`.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: It calls*

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    // Pass filteredArray to your UI layer for display.
+  }
+}
+
+- (NSMutableArray<ABKContentCard *> *)getCardsForFeedType:(NSString *)type {
+  NSMutableArray<ABKContentCard *> *cards = [NSMutableArray arrayWithArray:[Appboy.sharedInstance.contentCardsController getContentCards]];
+
+  NSMutableArray<ABKContentCard *> *filteredArray = [NSMutableArray arrayWithArray:[cards filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(ABKContentCard * card, NSDictionary *bindings) {
+    NSDictionary *extras = [card extras];
+      if (extras != nil && [extras objectForKey:@"feed_type"] != nil && [[extras objectForKey:@"feed_type"] isEqualToString:type]) {
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: Extra tab in lines 56-59

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+On iOS, this feature is currently only available for fully customized UIs. Contact your Customer Support Manager if you would like to display multiple Content Card feeds using our default UI.
+{% endalert %}
+
+The following documentation demonstrates an example implementation that can be changed to fit your specific integration.
+
+## Step 1: Setting Key-Value Pairs on Cards
+
+When creating a Content Card campaign, key-value pair data can be set on each Card. Our filtering logic will use this key-value pair data to categorize cards.
+
+For the purposes of this example, we'll set a key-value pair with the key `feed_type` that will designate which Content Card feed the card should be displayed in. The value will be whatever your custom feeds will be, as in `Transactional`, `Marketing`, and more.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: How about "as in `Transactional`, `Marketing`, etc."

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: fixed

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: nice, fixed

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md (hunk 3 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: leaving as is, its pattern matched from other platforms

**Comment by Bucimis** [comment]
@EmilyNecciai please merge :)

**Comment by hokstuff** [comment]
This branch was continued in [this other branch](https://github.com/Appboy/braze-docs/pull/608), so we no longer need the original

**Pull request closed without merging**